docs/UserGuide.md

```diff
@@ -66,53 +73,54 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message directing the user to our user guide.
 
+<!---
 ![help message](images/helpMessage.png)
+--->
 
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a resident : `add`
 
-Adds a person to the address book.
+Adds a resident to the database.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL f/FLOOR u/UNIT [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/John Doe p/98765432 f/5 u/88` adds a resident named John Doe, with a phone number, floor and unit number.
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com f/12 u/10 p/1234567 t/criminal` adds a resident named Betsy Crowe, with a phone number, floor and unit number, and 2 tags, *friend* and *criminal*.
 
-### Listing all persons : `list`
+### Listing all residents : `list`
 
-Shows a list of all persons in the address book.
+Shows a list of all residents in the RC4HDB database.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Editing an existing resident : `edit`
 
-Edits an existing person in the address book.
+Edits the data of an existing resident in the RC4HDB database.
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+Format: `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [f/FLOOR] [u/UNIT] [t/TAG]…​`
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
+* Edits the resident at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
Edits the resident who has the specified `MATRIC_NO`.

Reply:
Let's wait for team input.

Reply:
Let's revert back to using INDEX, this change should no longer be necessary :)